### PR TITLE
Register maurom.is-a.dev

### PR DESCRIPTION
**Pull request by MM-coder**
Register maurom.is-a.dev with A record pointing to 185.199.108.153.

### CODE REVIEW

File: domains/maurom.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+        "owner": {
+           "username": "MM-coder",
+           "email": "mauromartins@email.com",
+           "discord": "804668335039053874"
+        },
+    
+        "record": {
+            "A": ["185.199.108.153"]
```
Comment (DEV-DIBSTER) [inline domains/maurom.json:9]: Unable to view site, are you able to provide a preview?

**Review by DEV-DIBSTER — CHANGES_REQUESTED** [review]

File: domains/maurom.json (hunk 1 above)
Comment (MM-coder) [reply domains/maurom.json:9]: Hello, that's the github pages IP, current site is https://maurom.dev

Comment (CuteDog5695) [inline domains/maurom.json:9]: For it to work you need to add 3 more A records
```suggestion
        "A": [
            "185.199.108.153",
            "185.199.109.153",
            "185.199.110.153",
            "185.199.111.153"
        ]
```

**Review by iostpa — CHANGES_REQUESTED** [review]

**Comment by orxngc** [comment]
Closing as stale.

**Pull request closed without merging**